docs/src/components/panel.js

```diff
@@ -27,7 +27,7 @@ const Summary = styled.summary`
     content: '↓ show';
   }
 
-  ${Details}[open] &:after {
+  ${Details} [open] &:after {
```

Comment:
@dagoof probably worth fixing this sooner rather than later? @dalehurwitz this is a prettier/styled-components bug—shouldn't change here so you'll need to manually revert this one...

Reply:
yep. I'll fix now

Reply:
https://github.com/govau/notify/pull/156